crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -1769,6 +1836,7 @@ fn max_empty_lines(contents: &str) -> u32 {
         }
     }
 
+    max_new_lines = newlines.max(max_new_lines);
```

Comment:
This omission looks like a bug.

Reply:
If the _only_ tokens in the contents were newlines, we'd _never_ set `max_new_lines` -- we'd return 0!

Reply:
Ah yeah, I noticed that once too but It blew up all the tests when I tried to fix it